Dockerfile

```diff
@@ -91,6 +81,16 @@ RUN pip install --no-cache-dir -r requirements.txt
 COPY package.json /code
 RUN npm install
 
+
+FROM acousticbrainz-base AS acousticbrainz-dev
+
+COPY . /code
```

Comment:
Do we even need this for dev. because the dir is a volume, and this command won't do anything afaik.

Reply:
You're right, code isn't needed in dev. I only added it because I didn't know if you could have a `FROM` line that has no other commands. In this case we'd only use it to name the target.
Eventually I'd like to add test-specific python installation in this section, but didn't do it now because it's just a demo.

Reply:
Actually, I also use the acousticbrainz-dev target in jenkins, which doesn't have a volume. What do you think? Either we have a specific acousticbrainz-test target which COPYs the code, or we just leave this as-is.

I would prefer to use a -test or -dev target in jenkins so that we can install pytest _only_ during test, and not have it in the production image. What do you think?

Reply:
I made the change to install python packages for testing only in the -dev image, and we continue to use this image with jenkins (some conflicts, but I'll fix these before merging).